test/EFCore.PG.FunctionalTests/Query/ArrayArrayQueryTest.cs

```diff
@@ -297,11 +296,11 @@ public void Any_Contains()
             Assert.Empty(results);
 
             AssertSql(
-                @"SELECT s.""Id"", s.""SomeArray"", s.""SomeByte"", s.""SomeByteArray"", s.""SomeBytea"", s.""SomeMatrix"", s.""SomeText""
+                @"SELECT s.""Id"", s.""SomeArray"", s.""SomeByte"", s.""SomeByteArray"", s.""SomeBytea"", s.""SomeMatrix"", s.""SomeStringArray"", s.""SomeText""
 FROM ""SomeEntities"" AS s
-WHERE (ARRAY[2,3]::integer[] && s.""SomeArray"")",
-                //
-                @"SELECT s.""Id"", s.""SomeArray"", s.""SomeByte"", s.""SomeByteArray"", s.""SomeBytea"", s.""SomeMatrix"", s.""SomeText""
+WHERE (ARRAY[2,3]::integer[] && s.""SomeArray"")"
+            );
+            AssertContainsInSql(@"
```

Comment:
Can you please revert this? In general I try to just use full assertion replacement (as everywhere else in this test suite), even if it means a bit more work when they change.